components/trial-bar/button.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from '~/components/button';
+import { TRIAL_BAR_BUTTON } from '../../utilities/constants';
+
+// This component accepts the same props as Button.
+// eslint-disable-next-line react/forbid-foreign-prop-types
+const { propTypes } = Button;
```

Comment:
I wonder if we really need this and whether it will break people that are using the `babel-plugin-transform-react-remove-prop-types` plugin to remove props. Would it be sufficient to just have a link pointing to the Dropdown component?

Reply:
I used `BuilderHeader` for reference, if that's working for the consumers I don't think this is much different. I will update it if need be.

Reply:
Please see https://github.com/salesforce/design-system-react/blob/master/components/builder-header/nav-dropdown.jsx#L16

Reply:
Ok - let's leave as is then.